Kernel/Memory/PageDirectory.cpp

```diff
@@ -52,16 +52,17 @@ ErrorOr<NonnullLockRefPtr<PageDirectory>> PageDirectory::try_create_for_userspac
     }
 #endif
 
+#if ARCH(I386) || ARCH(X86_64)
```

Comment:
What does this change to PageDirectory achieve? if anything it deserves a comment, but a commit with a descriptive commit message would be better.